libkodimote/libkodimote.pro

```diff
@@ -133,3 +133,8 @@ HEADERS += libkodimote_global.h \
            addonsource.h \
            profiles.h \
            profileitem.h
+
+maemo5 {
+    target.path = /opt/libkodimote/lib
+    INSTALLS += target
+}
```

Comment:
That's true. I just figured out, that the line was added by QtCreator/me on opening the project. QtCreator complains about missing deployment information if this part is missing and on choosing to add it automatically these lines are inserted.
But due to the statical linkage the deployment is not necessary, so you are right, the section should be removed (as it will also install `libkodimote.a` in `/opt`). I have just to remember to ignore QtCreator's warning. :)
BTW should `CONFIG += static` be changed to `CONFIG += staticlib`?  See http://qt-project.org/wiki/How_to_create_a_library_with_Qt_and_use_it_in_an_application#fcaa56d9b97ba0f5703da2272539579e